frontend/src/components/ChatBox/ChatBoxMessage/MessageBubble.tsx

```diff
@@ -34,11 +34,29 @@ function MessageBubble({
 		positionClassName
 	);
 
+	const messageAuthor =
+		message.type === CHAT_MESSAGE_TYPE.LEVEL_INFO
+			? ''
+			: message.type === CHAT_MESSAGE_TYPE.USER
+			? 'You said: '
+			: message.type === CHAT_MESSAGE_TYPE.USER_TRANSFORMED
+			? 'Your message transformed by XML tagging: '
+			: message.type === CHAT_MESSAGE_TYPE.ERROR_MSG
+			? 'Error message: '
+			: message.type === CHAT_MESSAGE_TYPE.BOT
+			? 'ScottBrewBot said:  '
+			: 'ScottBrewBot said: ';
```

Comment:
Tiny, but seeing as these are almost the same, we could simplify this. Unless the extra space is critical!
```suggestion
			: 'ScottBrewBot said: ';
```

Reply:
I thought it was, but turns out it's not!